MediaGalleryUi/Test/Mftf/Test/AdminStandaloneMediaGallerySearchByKeywordTest.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminStandaloneMediaGallerySearchByKeywordTest">
```

Comment:
This test depends on Adobe Stock functionality, let's move it to AdobeStockImageAdminUi module